larray/tests/test_array.py

```diff
@@ -4134,6 +4134,16 @@ def test_dump():
                    ['a1', 1],
                    ['a2', 2]]
 
+    arr = ndtest((Axis('a0,a1'), Axis(2, 'b')))
```

Comment:
add a (short) comment to tell what you are testing. ie, something like: "with an anonymous axis"